src/main/js/pathUtil.ts

```diff
@@ -5,3 +5,24 @@ export function pathWithTrailingSlash(path: string) {
   }
   return path;
 }
+
+export function isValidPath(path: string): boolean {
+  const allowlistCharactersInPath = /^([\w.\-_/ ]+)$/;
```

Comment:
Wouldn't it be better to use a better regex that contains the extra checks below? 

For example: `^(?!-)[a-zA-Z0-9-]+(?<!-)(/(?!-)[a-zA-Z0-9-]+(?<!-))*$`

Reply:
You can check the details of the regex here: https://regex101.com/

Reply:
Regex.com says this regex may not be supported in all browsers. So i used another one.